hydra-redis/classes_objects.py

```diff
@@ -0,0 +1,118 @@
+import urllib.request as ur
+import json
+from redisgraph import Node, Edge
+
+
+class Classes:
+    """Contains all the classes endpoint and the objects"""
+
+    def __init__(self, redis_graph):
+        self.redis_graph = redis_graph
+
+    def objects_property(self, objects_node, new_list, url_node, api_doc):
+        """Makes nodes for those properties which have properties in itself."""
+        print("for the properties which have class and should have node")
+        print(new_list)
+        for obj in new_list:
+            obj_method = []
+            obj_prop = []
+            properties = []
+            obj = obj.replace("vocab:", "")
+            objects_properties = {}
+            for obj1 in api_doc.parsed_classes[
+                                               obj][
+                                               "class"].supportedOperation:
+                obj_method.append(obj1.method)
+            print("classes", obj_method)
+            objects_properties["methods"] = str(obj_method)
+            # all the operations for the object is stored in method
+            for obj1 in api_doc.parsed_classes[obj]["class"].supportedProperty:
+                properties.append(obj1.title)
+                if obj1.prop.replace("vocab:", "") in api_doc.parsed_classes:
+                    obj_prop.append(obj1.prop.replace("vocab:", ""))
+            objects_properties["property"] = str(properties)
+            # all property which itself is an object
+            obj_properties_classlist = obj_prop
+            objects_properties["class_property"] = str(
+                obj_properties_classlist)
+            print(objects_node.alias + str(obj))
+            obj_alias = str(objects_node.alias + str(obj)).lower()
+            # key for the node of the object
+            print(obj_alias)
+            obj_node = Node(
+                label="object",
+                alias=obj_alias,
+                properties=objects_properties)
+            self.redis_graph.add_node(obj_node)
+            edge = Edge(objects_node, "has" + str(obj), obj_node)
+            self.redis_graph.add_edge(edge)
```

Comment:
Write separate functions for Redis. Like there can be functions to add edge and add node etc.